### PR TITLE
[Ready to merge] Update IRC channels to only mention #ruby

### PR DESCRIPTION
**Pull request by jhass**
#ruby-lang was merged into #ruby, let's update the site to reflect that.

----
*The below is no longer relevant, the discussion emerging from it caused #ruby-lang to be merged into #ruby*

The #ruby channel has four times as many users
than #ruby-lang, it makes no sense to steer new
users of the language to a place where they're
less likely to get help.

I'm active in both channels for a couple of years now,
the above is not simply based on the numbers, it's experience.

Let's base the claim on some hard numbers from a couple of minutes ago anyway:

* Users in #ruby: 915
* Users in #ruby-lang: 330
* Users that are in both channels: 146

That makes 184 unique users for #ruby-lang and 769 unique users in #ruby.

For new users who wish a quieter place and that are intimidated by a large number of listeners, we can consider advertising ##new2ruby a bit more, which is an environment specifically crafted for this kind of users.

### CODE REVIEW

File: ja/community/index.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ lang: ja
   * 最近は上述の#ruby-jaがよく利用されています。
 
 * freenodeの[#ruby](irc://freenode/%23ruby?charset=utf-8),
-  [#ruby-lang](irc://freenode/%23ruby-lang?charset=utf-8)
+  [#ruby](irc://freenode/%23ruby-lang?charset=utf-8)
```
Comment (yous) [inline ja/community/index.md]: This link still points to #ruby-lang.

Comment (jhass) [inline ja/community/index.md]: Good catch, thanks.

File: ja/community/index.md [hunk 2]
```diff
@@ -34,7 +34,6 @@ lang: ja
   * 最近は上述の#ruby-jaがよく利用されています。
 
 * freenodeの[#ruby](irc://freenode/%23ruby?charset=utf-8),
```
Comment (yous) [inline ja/community/index.md]: We can remove this comma.

**Comment by jhass** [comment]
Mh, I expected at least some discussion. So who do I need to ping to get at least a clear yay or nay?

**Comment by skade** [comment]
I think @thezen

**Comment by skade** [comment]
Sorry, wanted to delete a comment. I think most moderators of #ruby-lang are not very active on GH. Afaik, http://twitter.com/the_zenspider has a lot to do with #ruby-lang,

**Comment by zenspider** [comment]
The real problem is freenode itself.

_Technically_ (read in comic book guy voice)... `#ruby-lang` is the "official" channel and is more curated than `#ruby`. `#ruby-lang` has active ops and (afaik) a present owner (@dblack is the founder, but I believe has relinquished ownership to @drbrain). The problem is that the owner/founder of `#ruby` is awol and only has one (active) op (@apeiros, in .ch, so 24hr coverage is impossible). What I'd like to see is for the channels to be merged to `#ruby-lang` and set up `#ruby` as a redirect. At least that way we can maintain an active pool of ops, keep the spammers out, etc.

For some reason, the mods of freenode won't merge the channels. The reasons why have not been clearly given. @drbrain might be able to provide more insight.

**Comment by jhass** [comment]
We (well, @apeiros) actually remedied this situation recently, we got @zzak as founder of #ruby now and in turn @apeiros. Current access list:

```
1     sepp2k                 +Aiotv [modified ? ago]
2     fflush                 +ARefiorstv [modified 3w 1d 1h ago]
3     apeiros                +AFRefiorstv [modified 3w 1d 2h ago]
4     banisterfiend          +Aiotv [modified 2y 39w 2d ago]
5     seanstickle            +Aiotv [modified 2y 39w 2d ago]
6     Mon_Ouie               +Aiotv [modified 2y 39w 2d ago]
7     zzak                   +AFRefiorstv [modified 3w 1d 2h ago]
8     Radar                  +AFRefiorstv [modified 3w 0d 22h ago]
9     Havenn                 +Aiotv [modified 3w 1d 1h ago]
10    jhass                  +Aiotv [modified 3w 1d 1h ago]
11    *!*@freenode/staff/*   +ARefiorstv [modified 3w 0d 22h ago]
12    Karpah                 +Aiotv [modified 2w 6d 15h ago]
13    miah                   +Aiotv [modified 2w 6d 3h ago]
14    workmad3               +Aiotv [modified 2w 2d 9h ago]
15    Coraline               +Aiotv [modified 1w 4d 4h ago]
```

Which gives us pretty good ops coverage. We're also thinking about exploring possibilities of channel self-regulation (enough users requesting a ban from a bot does a temporary ban and stuff like that).

**Comment by apeiros** [comment]
Since I got pinged, let me say my 2 cents too: I was unhappy with the situation of #ruby - the founder being on the channel about 5 times in 10 years, me being almost entirely the only active operator over those 10 years, and not having sufficient privileges to change silly things like the channel welcome message still linking to the no longer existing modruby-for-apache website.
Now that I have founders access, it's my intent to clean up #ruby's (the channel's) act. Getting more ops was the first step, fixing tiny things like the welcome message the next. We're now working on our tooling.
Personally, I have not made up my mind about #ruby-lang vs. #ruby yet. I first want to see what comes from our efforts to improve #ruby. For the time being, I have no issues with either #ruby or #ruby-lang being officially linked.

**Comment by drbrain** [comment]
It seems that most people have decided #ruby is the official channel.  I like this change.  I would like to defer to @apeiros on when this change should be applied.

A few times I tried to get Freenode to officially recognize #ruby-lang. Last I checked they no longer offered that.

I don't think changing the ruby-lang website will have any effect on which channel people join.

**Comment by apeiros** [comment]
IMO we can change the website immediately. I concur with you @drbrain, that the change of the website won't change much about the joining pattern. With regards to the decision (and potentially implementation) of merging #ruby and #ruby-lang, I don't think we need to hurry.

**Comment by jhass** [comment]
Thanks @drbrain and @apeiros!

@skade Since you participated here already, would you mind to do the merge? :)

**Comment by zzak** [comment]
LGTM

**Comment by zenspider** [comment]
I would like to see the merge and redirect happen. It'll reduce confusion and provide more consistent coverage/help for newbs (who are the real ones who suffer from this current mess).

**Comment by zzak** [comment]
@zenspider Sure, but who can setup the redirect?

**Comment by apeiros** [comment]
@zzak @drbrain can set up the redirect. There's also a tiny bit of politics to clear. If you and @radar agree, I'd say you grant @dbrain founder access on #ruby, and we add all #ruby-lang ops to #ruby.

**Comment by apeiros** [comment]
Procedure for redirect:
* /mode #ruby-lang +if #ruby (@drbrain needs at least ops in #ruby to do that)
* kick all users from #ruby-lang (I can let ruboto do that, or we can ask #freenode staff)

Command to grant @drbrain founder access: /cs flags #ruby add drbrain +AFRefiorstv

**Comment by zzak** [comment]
I've granted @drbrain founder access of #ruby:

> Flags +AFRefiorstv were set on drbrain in #ruby.

**Comment by zzak** [comment]
Any other operators we should add to #ruby? /cc @zenspider

**Comment by apeiros** [comment]
I've added all #ruby-lang operators as ops to #ruby.

**Comment by aredridel** [comment]
I've been an op of #ruby-lang for a while, though not present as much lately due to a lot of work in node.js rather than ruby. Happy to stay on the list though.

**Comment by djberg96** [comment]
Yep, let's merge the channels.

**Comment by drbrain** [comment]
@hsbt @stomar can you merge this?

**Comment by dblack** [comment]
FWIW, I'm all for the merger. I've actually been much less enthusiastic
about IRC since another 'dblack' poached my username. (Not really his
fault; the FreeNode people arranged for him to do it because I'd been away
for some number of months.)


David

**Comment by stomar** [comment]
Nice. It really _was_ confusing.

**Pull request merged**